storyblok/storyblok-docs
# Redirect to V2 for the default route

This update adds a `vercel.json` config file for redirects. There's a redirect set from `/docs/api/content-delivery/` to `/docs/api/content-delivery/v2/` so users are more aware of which version of the docs it is. The redirect is a 302 instead of a 301 so it's not cached by the browser.

The update to the `docgen` file is to generate also the routes for the `v1` docs which was previously not generated as it was converted to the route without the `/vX` part of the URL.

### How to test
Visit `https://storyblok-docs-q7zxbeg1f-storyblok-com.vercel.app/docs/api/content-delivery/` you should be redirected to `https://storyblok-docs-q7zxbeg1f-storyblok-com.vercel.app/docs/api/content-delivery/v2/`. Try also to visit the main route and `https://storyblok-docs-q7zxbeg1f-storyblok-com.vercel.app/docs/api/content-delivery/v1/` to make sure it's all good.
